test.js

```diff
@@ -166,6 +167,13 @@ test(`use relative path with '..' chars`, async t => {
 	t.is(stdout, 'foo');
 });
 
+test('execa() throws if running non executable', async t => {
+	// windows systems do not know executable-flags
+	if (os.platform() !== 'win32') {
```

Comment:
Better to wrap the whole test in that if statement. Also no need to import `os`. Just use `process.platform`.